scanner/updater/export.go

```diff
@@ -51,7 +52,9 @@ func Export(ctx context.Context, outputDir string, opts *ExportOptions) error {
 	bundles["nvd"] = nvdOpts()
 	bundles["epss"] = epssOpts()
 	bundles["stackrox-rhel-csaf"] = redhatCSAFOpts()
-	bundles["cisa-kev"] = kevOpts()
+	if features.CISAKEV.Enabled() {
```

Comment:
@dcaravel I updated the PR by just adding this feature flag conditional since you last reviewed. I'm going to assume you are ok with this and merge once CI passes

Reply:
OK by me - for consideration, by making it a feature flag it will be captured via telemetry and returned to the UI in response to `https://$ROX_ENDPOINT/v1/featureflags` - given that this setting is not used by live instances - could save some space in both telemetry and data returned to the UI by using the env package instead.